src/load.ts

```diff
@@ -352,6 +357,38 @@ export function loadFromPath(
   }
 }
 
+export async function lookUpLatestVersion(
+  packageName: string,
+  log: LogFunction = () => {}
+): Promise<string> {
+  const customRegistry = getCustomRegistry(log);
+  let res: AxiosResponse;
+  if (customRegistry) {
+    res = await axiosGet(`${customRegistry.replace(/\/$/, '')}/${packageName}`, {
+      responseType: 'json'
+    });
+  } else {
+    try {
+      res = await axiosGet(`https://packages.fhir.org/${packageName}`, {
+        responseType: 'json'
+      });
+    } catch (e) {
+      // Fallback to trying packages2.fhir.org
+      res = await axiosGet(`https://packages2.fhir.org/packages/${packageName}`, {
+        responseType: 'json'
+      }).catch(() => {
+        // If the fallback failed too, just throw the original error
+        throw e;
```

Comment:
This is just a question, but do you think this should throw something like the `PackageLoadError`? For example, when I couldn't connect to packages.fhir.org, I got the nice "The package hl7.fhir.us.mcode#2.0.0 could not be loaded locally or from the FHIR package registry." message for a package that had a version number specified, but I got the basic "getaddrinfo ENOTFOUND packages.fhir.org" error message for a package that specified `lastest` as the version, which I think comes from this line where we just throw the existing error.

Honestly, I'm not sure if this is obscuring helpful information or providing a more user-friendly message, so I'm very open to hearing what you think. It looks like when I try with SUSHI, the package with latest does still using the nicer package loading error, but the automatic dependencies fail to load and give the same "getaddrinfo ENOTFOUND packages.fhir.org" message, so there's a mix.

Reply:
Taking a look at the SUSHI implementation, it looks like if the check for what the latest version fails, it throws an error with `Could not determine latest released version`, which is then immediately caught and logged along with a `Failed to load` message. So, I think here, rather than just rethrowing, I can have it throw an error similar to that first one.